packages/cli-ui/src/hooks/settings.hook.ts

```diff
@@ -0,0 +1,33 @@
+import { useState, useCallback, useEffect } from 'react'
+
+import i18n from '../i18n'
```

Comment:
@VladKalachev @goldLifter  Думаю добавить allias `@i18n`, как идейка?
```suggestion
import i18n from '@i18n'
```

Reply:
можно, раз allias у нас используются в других местах кода